packages/next-sanity/src/visual-editing/index.tsx

```diff
@@ -7,9 +8,26 @@ const VisualEditingComponent = lazy(() => import('next-sanity/visual-editing/cli
  * @public
  */
 export function VisualEditing(props: VisualEditingProps): React.ReactElement {
+  // detected basePath, its value, you can change it by setting the basePath prop on VisualEditing
+  // or disable the auto mode by setting basePath to false
```

Comment:
You usually write such well-crafted comments, so I thought I would bring your attention to this one which I found to be a bit cryptic.

Reply:
Good catch! That was a note to myself that I was supposed to delete before opening the PR 😱